sonar-plugin-api/src/test/java/org/sonar/api/server/rule/RulesDefinitionXmlLoaderTest.java

```diff
@@ -57,6 +57,9 @@ public void parse_xml() throws Exception {
     assertThat(rule.severity()).isEqualTo(Severity.BLOCKER);
     assertThat(rule.template()).isTrue();
     assertThat(rule.status()).isEqualTo(RuleStatus.BETA);
+    assertThat(rule.debtSubCharacteristic()).isEqualTo("INTEGRATION_TESTABILITY");
+    assertThat(rule.debtRemediationFunction().coefficient()).isEqualTo("2d 2h 2min".replaceAll(" ", ""));
+    assertThat(rule.debtRemediationFunction().offset()).isEqualTo("1d 1h 1min".replaceAll(" ", ""));
```

Comment:
Corrected.
I decided to add that .replaceAll(" ", "") as notification that parser during rules uploading is able to validate input like "2d 2h 2min" but value is returned without blanks.